src/lib/support/tests/TestBufBound.cpp

```diff
@@ -56,19 +56,22 @@ class BBTest : public BufBound
             return false;
         }
 
-        if ((mLen < written && Fit()) || (mLen >= written && !Fit()))
+        size_t written;
```

Comment:
Suggest that tests where a variable is expected to be overwritten by an output parameter, you assign the variable to a known canary value that would also fail the further assertions below. Then you can catch code paths where the output parameter is not written as expected. This case is a bit trivial here, but I think it would apply, rather than relying on the luck of a given execution's stack contents for a default value.